src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/UnprotectedAttributeTests.cs

```diff
@@ -305,8 +305,28 @@ public static void TestUnprotectedAttributes1_EmptySet()
 
             EnvelopedCms ecms = new EnvelopedCms();
             ecms.Decode(encodedMessage);
-            
+
             Assert.Equal(2, ecms.UnprotectedAttributes.Count);
+            
+            CryptographicAttributeObjectCollection collection = ecms.UnprotectedAttributes;
+
+            int countOfAttr0 = collection[0].Values.Count;
+            int countOfAttr1 = collection[1].Values.Count;
+
+            Assert.True(countOfAttr0 == 0 || countOfAttr0 == 0);
```

Comment:
I assume you meant 

    Assert.True(countOfAttr0 == 0 || countOfAttr1 == 0); 

here